src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -763,6 +843,67 @@ def test_fw_binaryoperator_unknown(fort_writer, monkeypatch):
     assert "Unexpected binary op" in str(excinfo.value)
 
 
+def test_fw_binaryoperator_precedence(fort_writer):
```

Comment:
You may want to add the 'same precedence level but no associativity' examples in this test as well.

Reply:
Done, thanks.